content/en/project/Understanding_Learning_Trajectories_in_VRIT_Dynamic_Behavioral_and_Neural_Signatures_of_Inference/index.md

```diff
@@ -0,0 +1,64 @@
+# Tzu-Yun_Kung_project
+
+
+---
+type: "project" 
+date: "2025-06-14" 
+title: "Understanding Learning Trajectories in VRIT:Dynamic Behavioral and Neural Signatures of Inference"
+
+names: [Tzu-Yun Kung]
+
+github_repo: [https://github.com/PSY6983-2021/project_template](https://github.com/TYK0903/Tzu-Yun_Kung_project.git)
+
+
+tags: [inference, learning trajectories, python, fmri]
+
+
+summary: "This project aims to examine how the brain supports these two types of inference—active and passive—during the process of posterior belief integration. I appled trial-by-trial analysis to track participants’ learning trajectories over time. "
+
+
```

Comment:
the end of the header has to be `---`. 

This is not pure markdown. 

This header is parsed by code to properly list your project on the website gallery. 

Any deviation from the project guide will break the website.